src/component/PrintPanel/PrintPanelV3.tsx

```diff
@@ -40,7 +40,7 @@ interface DefaultPrintPanelV3Props {
 export interface PrintConfig {
   printServletPath(): string;
   getPrintFormats(): string[];
-  customPrintParams(): object;
+  customPrintParams?: object;
```

Comment:
Should this be `customPrintParams()?: object;`?

Reply:
The customPrintParams are handled as an object in the code, so i think this is right.